Pyrado/pyrado/environments/barrett_wam/natnet_client.py

```diff
@@ -120,6 +121,10 @@ class NatNetClient:
     This code is based on work from Boris Belousov and Pascal Klink.
     """
 
+    rigidBodyListener: Optional[Callable[[int, str, Vector3, Quaternion], NoReturn]]
```

Comment:
oh wow, you are really tackling that class. it has been passed forward for years without changing. actually the vast majority comes from the company doing OptiTrack